app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -47,23 +52,41 @@ Postgraduate teaching bursaries and scholarships are only available for the subj
 | **Maths**                                              | £27,000  | £29,000          |
 | **Physics**                                            | £27,000  | £29,000          |
 
- To be eligible for a bursary, you’ll need at least one of the following:
+### Bursaries
+
+You do not need to apply for a bursary. If you’re eligible, you’ll automatically receive it.
+
+To be eligible for a bursary, you’ll need at least one of the following:
```

Comment:
To avoid the repetition of 'a' can we say: 

"To be eligible for a bursary, you’ll need either a:

* bachelor’s degree class 2:2 (honours) or higher
* master’s degree
* PhD"

Reply:
I think this is more GDS style to talk about 'at least one of the following'? We definitely use this pattern across GIT a lot so I think it is OK (but will leave for @EllieNodder !)

Reply:
Candidates can also have more than one - so they can have both a bachelor's degree and a master's degree. So in this case, we need 'at least one' rather than 'either', but I see your point about repetition. Wondering if there's a way around this to still avoid repetition, any thoughts @Dom-Collins?